node/node.go

```diff
@@ -119,7 +121,11 @@ func NewNode(config cfg.Config, privValidator *types.PrivValidator, clientCreato
 	consensusReactor := consensus.NewConsensusReactor(consensusState, fastSync)
 
 	// Make p2p network switch
-	sw := p2p.NewSwitch(config.GetConfig("p2p"))
```

Comment:
don't think anybody was using this. looks like the config for p2p has to be in `p2p/config.toml` in order to be read.